src/utils.ts

```diff
@@ -76,3 +79,8 @@ export function releaseChildCanvases(el?: HTMLElement) {
     canvas.getContext('2d')?.clearRect(0, 0, 1, 1)
   })
 }
+
+// @internal
+export function isDocument(document: Source): document is PDFDocumentProxy {
```

Comment:
I think this can be simplified to the following, since the argument type doesn't matter and the return value is a boolean:

```suggestion
export function isDocument(document: unknown) {
```